README.md

```diff
@@ -1,14 +1,21 @@
 # Pathogenicity Preprocessing Workflow
-
+To run, please see the [CAVATICA app](https://cavatica.sbgenomics.com/public/apps/cavatica/apps-publisher/d3b-diskin-pathogenicity-preprocess-wf). Each version should correspond with a git release. This repo makes use of the git submodule feature for ease of code maintenance. To properly retrieve all relevant code:
+```sh
+git clone  https://github.com/d3b-center/D3b-Pathogenicity-Preprocessing
+git submodule init
+git submodule update
+```
 ## Prequisite
-It is recommended to have first run the [Kids First Germline Annotation Workflow](https://github.com/kids-first/kf-germline-workflow/blob/v1.0.0/docs/GERMLINE_SNV_ANNOT_README.md) first.
+It is recommended to have first run the [Kids First Germline Annotation Workflow](https://github.com/kids-first/kf-annotation-tools/blob/v1.1.0/docs/GERMLINE_SNV_ANNOT_README.md) first.
 
 ## Pathogenicity Preprocessing Workflow
 This workflow uses the prerequisite input to run the InterVar workflow and autoPVS1 tool.
 The major pieces of software being used are:
  - ANNOVAR latest: The software has no versioning, but references do. See `annovar_db` section in [Recommended inputs](#recommended-inputs)
  - InterVar v2.2.1
  - AutoPVS1 v2.0.0: Modified from AutoPVS1 v2.0 to fit annotated KF vcf output. See [README for autoPVS1](https://github.com/d3b-center/D3b-autoPVS1/tree/v2.0.0#readme) for details
+
+Optionally, if you which to add (and in needed, overwrite) another annotation from a vcf file (likely ClinVar), a bcftools strip and annotate steps are provided. The input vcf will be processed and its result will appear as an additional output in the workflow. 
```

Comment:
Optionally, if you wish to add (and overwrite if needed) another annotation from a vcf file (likely ClinVar), a BCFtools strip and annotate steps are provided. The input vcf will be processed and its result will appear as an additional output in the workflow.